Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -277,8 +279,8 @@ final class VisibleItemsProvider {
       case .day(let day):
         month = day.month
         calendarItemModel = self.content.dayItemModelProvider(day)
-      case .dayOfWeekInMonth:
-        return
+      case .dayOfWeekInMonth: return
+      case .monthFooter: return
```

Comment:
```suggestion
      case .dayOfWeekInMonth:
        return
      case .monthFooter(let _month):
        month = _month
        if let monthFooterItemModelProvider = self.content.monthFooterItemModelProvider(month) {
          calendarItemModel = monthFooterItemModelProvider
        } else {
          return
        }
```

Without this, we'll omit footers for people using VoiceOver and other accessibility technology.